config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -276,6 +276,9 @@ presubmits:
     - release-\d+\.\d+ # per-release image
     annotations:
       fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
```

Comment:
What policy are you trying to standardize to for this? Seems like something that could be enforced via tests against job configs